roles/keycloak/templates/configmap-certs.yml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: configmap-certs
+  namespace: keycloak
+
+data:
+  testbed-cert.pem: |
```

Comment:
Do not add testbed files here as hardcoded default. The value of the data dictionary has to be configurable via a parameter. This parameter has then to be added to the osism/testbed repository.

Reply:
File completely removed. Opted for a mounted volume with all certs into the initContainer